### ISSUE TITLE
Start new rollouts when secrets or configuration changes

### ISSUE THREAD

**Issue by csomh** [packit/deployment#331]
With Ansible it should be possible to register when a secret or configuration was changed in the cluster. Register these changes when they happen and force a new rollout of the deployments which depend on them.

This should save us the time to start rollouts manually, when all that a `make deploy` does is to update some secrets or configs.

**Comment by jpopelka** [comment]
Do you perhaps have some link(s)?
Is it something like [`notify` & `handlers`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html)?

**Comment by csomh** [comment]
> Do you perhaps have some link(s)? Is it something like [`notify` & `handlers`](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html)?

Indeed, it is. Thanks for providing the link!

**Comment by mfocko** [comment]
As for the current state of secrets, only the Sentry DSN (which we do not rotate that frequently) and Copr (which has an indefinite expiration right now) token cannot be used with the older secrets at the same time.

On Monday we did a quite seamless renewal of TLS certs, I renewed TLS, @lachmanfrantisek moved the production and redeployed. There was no need to immediately rescale to update to the latest tokens.

IMO currently the benefits do not outweigh the cost of adjusting the deployment. Therefore after the discussion on today's stand up, closing this issue for now. We can reopen if needed.
